src/components/layout/Layout.scss

```diff
@@ -4,3 +4,10 @@
   padding: 0;
   margin: 0 auto;
 }
+
+@media (max-width: 800px) {
```

Comment:
so from 1280 to 800 we will have content without any borders. You can do it higher without media queries

Reply:
so, as i understood correctly, i must delete  everything after @media and add padding:20px in .main? i tried to make @media without width, but there were errors.